ieeeuoft/newhacks-site-2022
# Added MLH banner, code of conduct and changed registration date

## Overview

- Added MLH banner and MLH logo to sponsors section with the Code of Conduct
- Changed registration open date to September 5th


## Unit Tests Created

- none


## Steps to QA

- go to newhacks landing page and make sure the MLH photos and links are correct
- make sure the registration open date is September 5th
